src/test/java/com/google/cloud/tools/gradle/appengine/AppEnginePluginTest.java

```diff
@@ -61,6 +62,7 @@ public void testCheckGradleVersion_fail() throws IOException {
                   + AppEngineCorePluginConfiguration.GRADLE_MIN_VERSION
                   + " or higher."));
     }
+    JavaVersion.current().isJava9Compatible();
```

Comment:
What is this for? Why?

Reply:
yeah that shouldn't be in there, must have been around from some old thing I was testing. I'll remove it